src/pkg/clouds/azure/aca/cert.go

```
@@ -137,23 +142,32 @@ func findContainerAppByService(ctx context.Context, client *armappcontainers.Con
 
 	for {
-		cnameOK := dns.CheckDomainDNSReady(ctx, hostname, []string{expectedCname}, resolverAt)
+		routeOK := dns.CheckDomainDNSReady(ctx, hostname, []string{expectedCname}, resolverAt)
 		txtOK, _ := dns.LookupTXTContains(ctx, asuid, expectedTxt, resolverAt(""))
-		if cnameOK && txtOK {
+		if routeOK && txtOK {
 			term.Infof("DNS records for %s verified", hostname)
 			return nil
 		}
 		if !promptShown {
 			term.Printf("Configure DNS records for %s:\n", hostname)
```

golangci-lint (2.12.2)

[error] Error return value of `term.Printf` is not checked

(errcheck)